documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+in feedback cycles over issues that could be addressed later on, and there were
+few, if any, expectations around who would participate in a discussion or what
+the expectations of that participation was. Adopting the process described
+below, we hope, will address some of the underlying causes of these problems, in
+addition to the benefits described above.
+
+The process described below is heavily modified from the regular Sociocratic
+process. The differences come from two directions:
+
+- The needs of a largely asynchronously coordinated base of contributors
+- Internal feedback and iteration on the initial process, even after
+  modification for asynchrony
+
+> ## When to follow this process
+>
+> Whether this process is followed for any given proposal or discussion is left
+> entirely to the discretion of the people relevant to the discussion. We
+> believe that this process helps the project's contributors make expedient and
+> clear decisions. However, it should not become an obstacle that is followed
+> for every tiny decision. If a decision can efficiently be made without this
+> process, contributors are invited to do so.
+>
+> For the purposes of project planning, however, this process should be the
+> default and should be the strong preference, keeping in mind that it can be
+> made faster via [round shortcutting](#shortcutting-a-round) and the
+> [synchronous lightening process](#optional-synchronous-lightening-process).
+
+## Continuous improvement and exceptions to every rule
+
+**This process is not set in stone.** Nor is it a set of laws that must be
+followed at all times. This document uses prescriptive language filled with
+imperatives primarily to prevent the process itself from being filled with
+contingencies at every turn. However, as with every other aspect of our ever
+evolving processes, these are merely suggestions that we expect contributors to
+adhere to. They cannot cover every situation and undoubtedly there are
+exceptional situations where following these suggestions to the letter would be
+antithetical to the goals of the project and even of the process itself.
+
+If during your time as an Openverse contributor, you feel there are ways in
+which this process can be improved and iterated upon, please know that this
+process should be iterated upon and that suggestions for improvement are open
+from any contributor to the project.
+
+## Terms
+
+- "Author": The person making the proposal.
+- "Participants": All participants of the decision-making process, including the
+  author, the requested participants, and ad-hoc participants.
+- "Reactions": A broad class of feedback that can include personal preferences
+  as well as general doubts about the validity or safety of the solutions
+  described in the proposal.
+- "Question": A request for clarification or explanation of an aspect of the
+  proposal. Questions may also request additional information be added to the
+  proposal.
```

Comment:
Would a suggestion to improve the project be considered a question (as in Can we add .../do it this way?)? One example is this suggestion comment: https://github.com/WordPress/openverse/pull/855#discussion_r1130233362

Reply:
I would hope so. "Would it make sense to try _x_ approach?" seems like a very reasonable question. Perhaps "Can you clarify why we wouldn't try _x_ instead?", etc, etc.

If the statement ends, grammatically, with an `?`, I presume it is fine. Are there specific types of questions you think we should not expect?

Reply:
Your reply and clarifications make total sense. Thank you for confirmation!